crates/engine/tree/src/tree/payload_processor/prewarm.rs

```diff
@@ -70,8 +70,7 @@ where
             pending,
             ctx,
             in_progress: 0,
-            // TODO settings
-            max_concurrency: 4,
+            max_concurrency: 64,
```

Comment:
can we add some context in the comment about how this value was determined, why 4 is too small and 64 isn't too big? probably something about EVM doing I/O and not pure CPU work, so Rayon's task per thread isn't the best idea here, but also why not 512 given that there's 512 blocking Tokio threads available